phenotrex/ml/feature_select.py

```diff
@@ -89,22 +52,31 @@ def recursive_feature_elimination(records: List[TrainingRecord], pipeline: Pipel
 
     logger = get_logger(__name__, verb=True)
     split = StratifiedKFold(shuffle=True, n_splits=5, random_state=random_state)
-    selector = RFECV(estimator, step=step, min_features_to_select=n_features, cv=split, n_jobs=5,
-                     scoring=DEFAULT_SCORING_FUNCTION)
+    selector = RFECV(
+        estimator,
+        step=step,
+        min_features_to_select=n_features,
+        cv=split,
+        n_jobs=5,
+        scoring=DEFAULT_SCORING_FUNCTION
+    )
     selector = selector.fit(X=X_trans, y=y)
 
     original_size = len(previous_vocabulary)
     support = selector.get_support()
     support = support.nonzero()[0]
     new_id = {support[x]: x for x in range(len(support))}
-    vocabulary = {feature: new_id[i] for feature, i in previous_vocabulary.items() if
-                  not new_id.get(i) is None}
+    vocabulary = {
+        feature: new_id[i]
+        for feature, i in previous_vocabulary.items()
+        if not new_id.get(i) is None
```

Comment:
```suggestion
        if new_id.get(i) is not None
```
Sounds more natural to me.